common/calibration-model.cpp

```diff
@@ -344,6 +346,7 @@ void calibration_model::update(ux_window& window, std::string& error_message)
         ImGui::SetCursorPosX(200);
 
         ImGui::PushItemWidth(120);
+        // cppcheck-suppress invalidContainer
```

Comment:
1) I don't think we should suppress checks for specific tools in the code, usually they can be suppressed using an option file as a parameter to the tool run

2) The problem here is real and should not be suppressed.
at [line 151](https://github.com/IntelRealSense/librealsense/pull/11397/files#diff-a585a9c9796db25f161ef05883ab5efb7e4be48e753ae693f46b670b3a521becR151) table is set to point to _calibration internal buffer. At  [line 302](https://github.com/IntelRealSense/librealsense/pull/11397/files#diff-a585a9c9796db25f161ef05883ab5efb7e4be48e753ae693f46b670b3a521becR302) the buffer is changed and the pointer is invalid. Setting table to the new buffer will solve the problem.

Reply:
done. Please verify